bears/python/PEP8Bear.py

```diff
@@ -31,14 +31,19 @@ def run(self, filename, file,
         Detects and fixes PEP8 incompliant code. This bear will not change
         functionality of the code in any way.
 
-        :param max_line_length:   Maximum number of characters for a line.
-                                  When set to 0 allows infinite line length.
-        :param indent_size:       Number of spaces per indentation level.
-        :param pep_ignore:        A list of errors/warnings to ignore.
-        :param pep_select:        A list of errors/warnings to exclusively
-                                  apply.
-        :param local_pep8_config: Set to true if autopep8 should use a config
-                                  file as if run normally from this directory.
+        :param max_line_length:
+            Maximum number of characters for a line
+            When set to 0 allows infinite line length.
```

Comment:
Max characters per line are set to 80 so this can be put with the above line.

Reply:
I think you misunderstood. I am saying that the second line can be with the first line like:
```
        :param max_line_length:
            Maximum number of characters for a line. When set to 0 allows
            infinite line length.
```

Reply:
Ya okay :sweat_smile:  but I think that is also correct as even after the changes you suggested we require two lines for writing description .
so I think that will be alright  Isn't it?

Reply:
> after the changes you suggested we require two lines for writing description.

Look at line 43

Reply:
okay but I think that thing doesn't affect the code and I think that is not necessary :wink:

Reply:
cc @jayvdb

Reply:
This one is OK, because they are two different sentences, and it would consume two lines anyway.